test/e2e/rdms/rdms_suite_test.go

```diff
@@ -0,0 +1,20 @@
+package rdms_test
+
+import (
+	"os/exec"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	clientv3 "go.etcd.io/etcd/client/v3"
+)
+
+var (
+	etcdClient         *clientv3.Client
+	etcdAdapterProcess *exec.Cmd
+)
+
+func TestRDMS(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "RDMS Suite")
```

Comment:
RDMS what's the meaning?

Reply:
+1

Reply:
`RDMS` is the `Relational Database Management System`

Now, I think `datasource` is more appropriate and I will modify it.

Reply:
I suggest using `RDBMS`.

Reply:
> I suggest using RDBMS.

@tokers Considering that we may also add support for NoSQL type databases in the future, I think datasource is a bit more appropriate.

Reply:
Right, agree with you 👍.